client/my-sites/email/email-providers-comparison/stacked/index.tsx

```diff
@@ -216,7 +219,17 @@ const EmailProvidersStackedComparison = ( {
 			<QueryProductsList />
 
 			{ ! isDomainInCart && selectedSite && <QuerySiteDomains siteId={ selectedSite.ID } /> }
-			{ isDomainInCart && <EmailUpsellNavigation siteSlug={ selectedSite?.slug ?? '' } /> }
+
+			{ ! hideNavigation && (
+				<EmailUpsellNavigation
+					backUrl={
+						isDomainInCart
+							? domainAddNew( selectedSite?.slug )
+							: emailManagement( selectedSite?.slug, null )
+					}
+					skipUrl={ isDomainInCart ? `/checkout/${ selectedSite?.slug }` : '' }
```

Comment:
I was about to suggest using some constant or function as we do in many sites (like the above lines: `emailManagement`). But after researching I couldn't see anything like that 😞 

Feels strange to me that we don't have a constant or checkout controller or any related function to handle those transitions between pages.

So... All fine!

Reply:
I see what you mean. This was based on what we had in `EmailUpsellNavigation` before, but would've been a good opportunity to update if there was some function for generating the path.